src/amr/messengers/hybrid_hybrid_messenger_strategy.h

```diff
@@ -449,6 +465,7 @@ namespace amr
 
             magneticGhosts_.fill(hybridModel.state.electromag.B, levelNumber, initDataTime);
             electricGhosts_.fill(hybridModel.state.electromag.E, levelNumber, initDataTime);
+            // currentGhosts_.fill(hybridModel.state.J, levelNumber, initDataTime); //TODO?
```

Comment:
is there an issue for this TODO?

Reply:
nope. I think it was a question for myself whether or not to do this here.
actually, this function is called here 

https://github.com/nicolasaunai/PHARE/blob/advancement/src/solver/level_initializer/hybrid_level_initializer.h#L60

i.e. just after the model initializes the root level.
However the model only loads the particles in each pop and initializes the magnetic field:

https://github.com/nicolasaunai/PHARE/blob/advancement/src/solver/physical_models/hybrid_model.h#L72-L79

so when the fillRootGhosts() is called, the current is actually 0, thus filling ghosts is useless.

The current J is now calculated in the hybrid level initializer here

https://github.com/nicolasaunai/PHARE/blob/advancement/src/solver/level_initializer/hybrid_level_initializer.h#L102-L116

mainly to be used by Ohm to get the electric field initialized on root level


long story short : I have deleted that line.

Reply:
> long story short : I have deleted that line.

lmao